src/components/CheckBoxFilterContainer.ts

```diff
@@ -0,0 +1,183 @@
+import { Component, ReactElement, createElement } from "react";
+import { findDOMNode } from "react-dom";
+
+import { Alert } from "./Alert";
+import { CheckboxFilter, CheckboxFilterProps } from "./CheckBoxFilter";
+import { Utils, parseStyle } from "../utils/ContainerUtils";
+
+import * as classNames from "classnames";
+import * as dijitRegistry from "dijit/registry";
+import * as dojoConnect from "dojo/_base/connect";
+import "./ui/CheckBoxFilter.scss";
+
+interface WrapperProps {
+    class: string;
+    style: string;
+    friendlyId: string;
+    mxform?: mxui.lib.form._FormBase;
+    mxObject: mendix.lib.MxObject;
+}
+
+export interface ContainerProps extends WrapperProps {
+    listviewEntity: string;
+    filterBy: filterOptions;
+    attribute: string;
+    attributeValue: string;
+    constraint: string;
+    defaultChecked: boolean;
+    unCheckedFilterBy: filterOptions;
+    unCheckedAttribute: string;
+    unCheckedAttributeValue: string;
+    unCheckedConstraint: string;
+}
+
+export type filterOptions = "attribute" | "XPath" | "None";
```

Comment:
Types, Classes and Interface should start with a Capital  `FilterOptions`
Dont think you need export it, as it is not used anywhere

Reply:
sure, thanks